pkg/ddc/juicefs/transform_fuse.go

```diff
@@ -411,3 +422,126 @@ func (j *JuiceFSEngine) genFormatCmd(value *JuiceFS, config *[]string) {
 	cmd := append([]string{common.JuiceCliPath, "auth"}, args...)
 	value.Configs.FormatCmd = strings.Join(cmd, " ")
 }
+
+func (j *JuiceFSEngine) getQuota(v string) (int64, error) {
+	q, err := resource.ParseQuantity(v)
+	if err != nil {
+		return 0, fmt.Errorf("invalid quota %s: %v", v, err)
+	}
+	qs := q.Value() / 1024 / 1024 / 1024
+	if qs <= 0 {
+		return 0, fmt.Errorf("quota %s is too small, at least 1GiB for quota", v)
+	}
+
+	return qs, nil
+}
+
+func (j *JuiceFSEngine) genQuotaCmd(value *JuiceFS, mount datav1alpha1.Mount) error {
+	options := mount.Options
+	for k, v := range options {
+		if k == "quota" {
+			qs, err := j.getQuota(v)
+			if err != nil {
+				return err
+			}
+			if value.Fuse.SubPath == "" {
+				return fmt.Errorf("subPath must be set when quota is enabled")
+			}
+			ceVersion, eeVersion, err := ParseImageTag(value.Fuse.ImageTag)
+			if err != nil {
+				return err
+			}
+			if value.Edition == CommunityEdition {
+				// ce
+				if ceVersion.LessThan(&ClientVersion{1, 1, 0, ""}) {
+					return fmt.Errorf("quota is not supported in juicefs-ce version %s", value.Fuse.ImageTag)
+				}
+				// juicefs quota set ${metaurl} --path ${path} --capacity ${capacity}
+				value.Configs.QuotaCmd = fmt.Sprintf("/usr/local/bin/juicefs quota set %s --path %s --capacity %d", value.Source, value.Fuse.SubPath, qs)
+				return nil
+			}
+			// ee
+			if eeVersion.LessThan(&ClientVersion{4, 9, 2, ""}) {
+				return fmt.Errorf("quota is not supported in juicefs-ee version %s", value.Fuse.ImageTag)
+			}
+			// juicefs quota set ${metaurl} --path ${path} --capacity ${capacity}
+			cli := "/usr/bin/juicefs"
+			value.Configs.QuotaCmd = fmt.Sprintf("%s quota set %s --path %s --capacity %d", cli, value.Source, value.Fuse.SubPath, qs)
+			return nil
+		}
+	}
+	return nil
+}
+
+func ParseImageTag(imageTag string) (*ClientVersion, *ClientVersion, error) {
+	if imageTag == "nightly" {
+		return &ClientVersion{0, 0, 0, "nightly"}, &ClientVersion{0, 0, 0, "nightly"}, nil
```

Comment:
if the string "nightly" is frequently used in the code, it should be defined as a constant. This will avoid redundant string constants in the code and make it easier to maintain and modify.

Reply:
done